include/swift/AST/TypeCheckRequests.h

```diff
@@ -2192,7 +2192,7 @@ class DerivativeAttrOriginalDeclRequest
 /// property in a `Differentiable`-conforming type.
 class TangentStoredPropertyRequest
     : public SimpleRequest<TangentStoredPropertyRequest,
-                           TangentPropertyInfo(VarDecl *),
+                           TangentPropertyInfo(VarDecl *, CanType),
```

Comment:
Note: adding a base type parameter to `TangentStoredPropertyRequest` results in less cache hits.
It may be possible to fix SR-13134 without adding a base type parameter, I may take a look later.